_1327/documents/views.py

```diff
@@ -44,7 +44,7 @@ def create(request, document_type):
 		model_class = content_type.model_class()
 		delete_old_empty_pages()
 		title = model_class.generate_new_title()
-		url_title = model_class.generate_default_slug(title)
+		url_title = "temp_{}".format(model_class.generate_default_slug(title))
```

Comment:
i would prefer not showing the `temp_` prefix in the form field, because users might always feel the need to remove that. could you maybe change the initial value in the form?